packages/medusa/src/api/store/products/helpers.ts

```diff
@@ -114,12 +114,6 @@ const asTaxItem = (product: HttpTypes.StoreProduct): TaxableItemDTO[] => {
       return {
         id: variant.id,
         product_id: product.id,
-        product_name: product.title,
-        product_categories: product.categories?.map((c) => c.name),
-        // TODO: It is strange that we only accept a single category, revisit the tax module implementation
-        product_category_id: product.categories?.[0]?.id,
-        product_sku: variant.sku,
-        product_type: product.type,
```

Comment:
These are currently not used for tax calculations so I removed them from the type and input.